### PR TITLE
Added Alexa support + new branding

### PR DESCRIPTION
**Pull request by dorsha**
image

image

### CODE REVIEW

File: web/router.go [hunk 1]
```diff
@@ -89,10 +89,10 @@ func handlePublicPath(pubPath string) {
 	// absolute path
 	case len(pubPath) > 1 && (pubPath[0] == '/' || pubPath[0] == '\\'):
 		public = pubPath
-	// absolute path win
+		// absolute path win
```
Comment (slavikm) [inline web/router.go]: Why the indentation?

File: web/securityhandlers.go [hunk 2]
```diff
@@ -2,15 +2,19 @@ package web
...
 import (
 	"context"
+	"encoding/base64"
 	"fmt"
 	"log"
 	"net/http"
 	"strings"
 	"time"
 
+	"math"
```
Comment (slavikm) [inline web/securityhandlers.go]: go-imports?

File: web/securityhandlers.go [hunk 3]
```diff
@@ -175,3 +179,91 @@ func (ac *AppContext) googleLogin(w http.ResponseWriter, r *http.Request) {
...
+		WriteError(w, ErrMissingPartRequest)
+		return
+	}
+
+	if clientID != conf.Options.Alexa.ClientID || !strings.HasPrefix(redirectURI, "https://pitangui.amazon.com") || responseType != "token" || scope != "demisto_alexa_skill" {
+		WriteError(w, ErrAuth)
+		return
+	}
+
+	ac.removeState(state)
```
Comment (slavikm) [inline web/securityhandlers.go]: Do we need this?

File: web/router.go (hunk 1 above)
Comment (dorsha) [reply web/router.go]: this is go fmt auto fix.

File: web/securityhandlers.go (hunk 2 above)
Comment (dorsha) [reply web/securityhandlers.go]: yes..

File: web/securityhandlers.go (hunk 3 above)
Comment (dorsha) [reply web/securityhandlers.go]: no will remove

**Pull request merged**